source/MaterialXTest/ShaderValid.cpp

```diff
@@ -495,6 +519,12 @@ static void runOSLValidation(const std::string& shaderName, mx::TypedElementPtr
             }
             validator.setOslShaderOutputName(outputName);
 
+            // Set scene template file. For now we only have the constant color scene file
```

Comment:
Will need a different scene setup for surface shading. Only have constant input for now though.